site/docs/concepts/web-app.md

```diff
@@ -93,11 +98,15 @@ All the collections of the selected capture(s) will be added to the materializat
 
 When you click **View Details** for a capture, the **Status** and **Specification** viewers are revealed.
 
-The **Status** window shows the full identifier of the shard that backs your capture. If there's an error, you can view its logs.
+The **Status** window shows the full identifier of the shard(s) that back your capture. If there's an error, you'll see an alert next to the shard identifier. Use the drop-down to open an expanded view of the failed shard's logs.
```

Comment:
I would also replace the instance of _window_ here with _section_.
<br />

>you'll see an alert next to the shard identifier.

The reference to the alert's positioning relative to the shard ID feels incorrect. I think someone would anticipate the alert appearing within the _Shard Information_ table, given that is where shard IDs are primarily surfaced. What about something a little more general: `If there's an error, you'll see an alert corresponding to the failing shard(s).` or `If there's an error, you'll see an alert identifying the failing shard(s).`